lib/uxtx.js

```diff
@@ -1,4 +1,4 @@
-const { Amount, TX } = require('bcoin');
+const { Amount, TX } = require('bcoin/lib/bcoin-browser');
```

Comment:
What is this change for?

Reply:
See the comment at the top. When using `require` webpack won't do the alias replacement and you have to manually enter the path to make sure to use the browser version of the library.

Reply:
Hmm, I'd like to figure out how to make it be able to work in node.js as well

Reply:
This change allows it to work in both, the browser target just loses the caching optimizations the non-browser entry point introduces.